<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Alex Peterson</author>
 <class>spacescape::MainWindow</class>
 <widget class="QMainWindow" name="spacescape::MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Spacescape - untitled.xml*</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../Spacescape.qrc">
    <normaloff>:/spacescape/resources/spacescape-logo.ico</normaloff>:/spacescape/resources/spacescape-logo.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <property name="leftMargin">
     <number>4</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QSplitter" name="splitter_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QtTreePropertyBrowser" name="layerProperties" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="headerVisible" stdset="0">
           <bool>false</bool>
          </property>
          <property name="resizeMode" stdset="0">
           <string>ResizeToContents</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="frame">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <property name="spacing">
            <number>5</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>3</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QPushButton" name="newLayerButton">
             <property name="statusTip">
              <string>Create a new layer</string>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>New Layer</string>
             </property>
             <property name="icon">
              <iconset resource="../Spacescape.qrc">
               <normaloff>:/spacescape/resources/page_white_add.png</normaloff>:/spacescape/resources/page_white_add.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="copyLayerButton">
             <property name="statusTip">
              <string>Copy the selected layer</string>
             </property>
             <property name="text">
              <string>Copy Layer</string>
             </property>
             <property name="icon">
              <iconset resource="../Spacescape.qrc">
               <normaloff>:/spacescape/resources/page_white_copy.png</normaloff>:/spacescape/resources/page_white_copy.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="moveLayerDownButton">
             <property name="statusTip">
              <string>Move the selected layer down</string>
             </property>
             <property name="text">
              <string>Move Down</string>
             </property>
             <property name="icon">
              <iconset resource="../Spacescape.qrc">
               <normaloff>:/spacescape/resources/page_white_put.png</normaloff>:/spacescape/resources/page_white_put.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="moveLayerUpButton">
             <property name="statusTip">
              <string>Move the selected layer up</string>
             </property>
             <property name="text">
              <string>Move Up</string>
             </property>
             <property name="icon">
              <iconset resource="../Spacescape.qrc">
               <normaloff>:/spacescape/resources/page_white_get.png</normaloff>:/spacescape/resources/page_white_get.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>62</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="deleteLayerButton">
             <property name="statusTip">
              <string>Delete the selected layer</string>
             </property>
             <property name="text">
              <string>Delete Layer</string>
             </property>
             <property name="icon">
              <iconset resource="../Spacescape.qrc">
               <normaloff>:/spacescape/resources/page_white_delete.png</normaloff>:/spacescape/resources/page_white_delete.png</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="spacescape::PreviewWidget" name="ogreWindow" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>100</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>256</width>
         <height>256</height>
        </size>
       </property>
       <property name="backgroundColor" stdset="0">
        <color>
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
    <addaction name="actionExport"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionShowDebugBox"/>
    <addaction name="actionEnableHDR"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menuSettings"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="text">
    <string>&amp;Open</string>
   </property>
   <property name="statusTip">
    <string>Open a file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="text">
    <string>S&amp;ave as...</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About Spacescape</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="icon">
    <iconset>
     <normaloff>images/page_white_add.png</normaloff>images/page_white_add.png</iconset>
   </property>
   <property name="text">
    <string>&amp;New</string>
   </property>
   <property name="statusTip">
    <string>Clear the scene and start over.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="text">
    <string>Export</string>
   </property>
   <property name="statusTip">
    <string>Export scene to six images for a skybox</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionEnableHDR">
   <property name="text">
    <string>Enable HDR</string>
   </property>
  </action>
  <action name="actionShowDebugBox">
   <property name="text">
    <string>ShowDebugBox</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QtTreePropertyBrowser</class>
   <extends>QWidget</extends>
   <header>QtTreePropertyBrowser.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>spacescape::PreviewWidget</class>
   <extends>QWidget</extends>
   <header>PreviewWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../Spacescape.qrc"/>
 </resources>
 <connections/>
 <slots>
  <slot>onExport()</slot>
  <slot>onNewFile()</slot>
  <slot>onNewLayerClicked()</slot>
  <slot>onCopyLayerClicked()</slot>
  <slot>onMoveLayerUp()</slot>
  <slot>onMoveLayerDown()</slot>
  <slot>onDeleteLayerClicked()</slot>
 </slots>
</ui>
